### DISCUSSION TITLE
Alternative app icon

### DISCUSSION THREAD

**Discussion by gordongw** [Docile-Alligator/Infinity-For-Reddit#1413] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

Please consider adding an alternative app icon, as the default one strikes one as less than elegant (no offense intended, the rest of the app is excellent).
Perhaps a simple "R" on a subdued background, or remaking the stock icon with fewer graphic elements (for example: no stars or planets in the background). Thanks.  

### Additional context

The default icon is somewhat kitsch. 

### How important is this feature?

Important

**Comment by ghost** [comment]
any updates on this?
